CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - **cmd/shfmt**
   - Add `-filename` to give a name to standard input
+  - Don't add parentheses to functions, if they were not in the original source
```

Comment:
the commit message and changelog should be about `syntax` instead, since that's where the change is being made (shfmt is just one of the consumers)

Reply:
Also, I would make this a little shorter and clearer:

```
- Don't add parentheses to `function {...}` declarations for ksh support
```

Reply:
Thanks!
As this is mostly about ksh support, adding "official" support for that would be good. I'll create a new issue for that. 
It's up to you if you want to merge this nontheless. I don't mind if this PR is dropped instead of ksh support.
Let me know if you want to merge this and I'll clean it up.

Reply:
I think merging this PR is fine anyway :) please do finish it up and I'll merge.

Reply:
@mvdan I've updated changelog and commit message.

Reply:
Thanks!